frontend/packages/dev-console/src/components/pipelineruns/status/PipelineBars.tsx

```diff
@@ -16,20 +16,11 @@ export interface PipelineBarProps {
 }
 
 export const PipelineBars: React.FC<PipelineBarProps> = ({ pipelinerun, pipeline }) => {
-  const pipelineIdentifier =
-    pipelinerun.metadata && pipelinerun.metadata.name ? pipelinerun.metadata.name : undefined;
-  const data = pipeline && pipeline.data ? pipeline.data : undefined;
-  const taskStatus = getTaskStatus(pipelinerun, data);
+  const taskStatus = getTaskStatus(pipelinerun, pipeline?.data);
 
-  // Tooltip key is necessary to disable the animations between two different Pipeline runs that
-  // have different names. Typically this should be able to go on HorizontalStackedBars but there
-  // is an issue with Tooltip and losing its mounted child node that breaks the popup.
   return (
-    <Tooltip key={pipelineIdentifier} content={<TaskStatusToolTip taskStatus={taskStatus} />}>
+    <Tooltip content={<TaskStatusToolTip taskStatus={taskStatus} />}>
       <HorizontalStackedBars
-        // Note: disableAnimation is used here to prevent a no-key situation where React will try to
-        // reuse elements and inadvertently cause animations BETWEEN different pipeline status'
-        disableAnimation={!pipelineIdentifier}
```

Comment:
Could you also delete the prop and usage inside `HorizontalStackedBars`?

Reply:
Done.